### DISCUSSION TITLE
How to use 3.0?

### DISCUSSION THREAD

**Discussion by Bluespide** [lepoco/wpfui#961] [Q&A] [answered]
Sorry, am I stupid or something? But with 3.0 I get the Error 'The name "UiWindow" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml"'. Is there a new namespace? Where is the UiWindow?

**Comment by johnvesalius** [comment, marked as answer]
No, you're not stupid. The README is somewhat outdated. `ui:UiWindow` is now `ui:FluentWindow`. Check out https://github.com/lepoco/wpfui/tree/main/src/Wpf.Ui.Demo.Simple for working code.

**Reply by Bluespide** [reply]
That was simple, thank you!
